x-pack/plugins/enterprise_search/public/plugin.ts

```diff
@@ -83,7 +83,6 @@ export class EnterpriseSearchPlugin implements Plugin {
       description: 'Search everything',
       path: '/app/enterprise_search/app_search', // TODO: update this link to enterprise search landing page
       order: 100,
-      className: 'homSolutionsSection__greenCard',
     });
```

Comment:
See comment above regarding why we likely need to keep this (or something similar to it).